src/morphometrics/segment/main.py

```diff
@@ -0,0 +1,136 @@
+import argparse
+import os
+from timeit import default_timer as timer
+
+import numpy as np
+import tifffile
+
+from ..segment import post_process, pre_process, prepare_image, segment
```

Comment:
You can import from the current directory with `.`. Also, you can import the specific functions you're interested in.

For example

```python
from .post_process import post_process
```